src/lib.rs

```diff
@@ -68,10 +68,11 @@ pub use endianity::{Endianity, EndianBuf, LittleEndian, BigEndian};
 
 mod parser;
 pub use parser::{Error, ParseResult, Format};
-pub use parser::{DebugStrOffset, DebugLocOffset, DebugMacinfoOffset, UnitOffset};
+pub use parser::{DebugLocOffset, DebugMacinfoOffset, UnitOffset};
 pub use parser::{DebugInfo, DebugInfoOffset, UnitHeadersIter, UnitHeader};
 pub use parser::{DebugTypes, DebugTypesOffset, TypeUnitHeadersIter, TypeUnitHeader};
 pub use parser::{EntriesCursor, DebuggingInformationEntry, AttrsIter, Attribute, AttributeValue};
+pub use parser::{DebugStrings, DebugStrOffset};
```

Comment:
We should use consistent names for `DebugStrings` and `DebugStrOffset`. I have a mild preference for `DebugStr` since that corresponds to the dwarf section name.